core/src/main/java/io/parsingdata/metal/expression/value/Value.java

```diff
@@ -55,10 +54,6 @@ public BitSet asBitSet() {
         return BitSet.valueOf(encoding.byteOrder == ByteOrder.BIG_ENDIAN ? ByteOrder.LITTLE_ENDIAN.apply(getValue()) : getValue());
     }
 
-    public OptionalValue operation(final ValueOperation operation) {
-        return operation.execute(this);
-    }
-
```

Comment:
This mechanism was needed to implement some operation on the data of a value. The `execute()` method would then handle evaluating, etc. and then pass the resulting `Value`. But that's what `ValueExpression` base classes like `UnaryValueExpression` and `BinaryValueExpression` also do, so no point in having this separate mechanism.